libnet/client.c

```diff
@@ -0,0 +1,137 @@
+#include "client.h"
```

Comment:
also "Pliki źródłowe powinny mieć w nagłówku metryczkę podającą w jednolity sposób co najmniej: nazwę projektu, autora (autorów), datę utworzenia." so we have to add it :////